driver-sync/src/test/functional/com/mongodb/client/AbstractTransactionsTest.java

```diff
@@ -322,8 +322,8 @@ private void executeOperations(final BsonArray operations, final boolean throwEx
 
                 ClientSession clientSession = receiver.startsWith("session") ? sessionsMap.get(receiver) : null;
                 if (clientSession == null) {
-                    clientSession = operation.getDocument("arguments").containsKey("session")
-                            ? sessionsMap.get(operation.getDocument("arguments").getString("session").getValue()) : null;
+                    clientSession = operation.containsKey("arguments") ? (operation.getDocument("arguments").containsKey("session")
```

Comment:
Instead of nested ternary, you can do this

     operation.getDocument("arguments", new BsonDocument()